content_api/app/settings.py

```diff
@@ -47,6 +47,7 @@ def env(variable, fallback_value=None):
 OAUTH2_TOKEN_URL = '/token'
 OAUTH2_SCOPES = ['content_api']
 BCRYPT_GENSALT_WORK_FACTOR = 12
+PUBLIC_RESOURCES = ['publish', 'assets', 'clients', 'users']
```

Comment:
but then maybe it would make sense to disable it until we can manage clients with ui